learn/security/tenant_tokens.md

```diff
@@ -0,0 +1,299 @@
+# Multitenancy and tenant tokens
+
+In this guide you'll first learn [what multitenancy is](#what-is-multitenancy) and [how tenant tokens help managing complex applications and sensitive data](#what-are-tenant-tokens-and-how-do-they-solve-multitenancy-in-meilisearch). Then, you'll see [how to generate and use tokens](#generating-tenant-tokens-with-an-sdk), whether with an official SDK or [otherwise](#generating-tenant-tokens-without-a-meilisearch-sdk). The guide ends with a quick explanation of [the main tenant token settings](#tenant-token-payload-reference).
+
+## What is multitenancy?
+
+In software development, multitenancy means that multiple users with different interests—also called tenants—share the same computing resources. Proper multitenancy is crucial in cloud computing services such as [Digital Ocean's Droplets](https://www.digitalocean.com/products/droplets) and [Amazon's AWS](https://aws.amazon.com/).
+
+If your Meilisearch application stores sensitive data like medical records in the same index, it is very important to make sure users can only search through their own documents.
```

Comment:
```suggestion
If your Meilisearch application stores sensitive data belonging to multiple users in the same index, it is very important to make sure users can only search through their own documents. This can be accomplished with **tenant tokens**.
```